helm/mongodb/Chart.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright VMware, Inc.
+# SPDX-License-Identifier: APACHE-2.0
```

Comment:
There's a lot of code in this mongo chart, is it just a lot of setup? Not too familiar with much of what's happening, so lmk if there are particular parts you want me to take a look at

Reply:
Yeah stuff could get cleaned up. Not too concerned since it works currently.